Source/NSURLConnection.m

```diff
@@ -452,6 +507,11 @@ - (void) URLProtocol: (NSURLProtocol *)protocol
 	initWithRequest: this->_request
 	cachedResponse: nil
 	client: (id<NSURLProtocolClient>)self];
+      
```

Comment:
Again, wouldn't calling the method start be more consistent here? E.g. that would set the new _started instance variable.